platform/ios/src/MGLMapView.h

```diff
@@ -286,12 +304,32 @@ MGL_EXPORT IB_DESIGNABLE
  */
 @property (nonatomic, readonly) UIView *scaleBar;
 
+/**
+ The position of the scale bar. The default value is `MGLOrnamentPositionTopLeft`.
+ */
+@property (nonatomic, assign) MGLOrnamentPosition scaleBarPosition;
+
+/**
+ A `CGPoint` indicating the position offset of the scale bar.
+ */
+@property (nonatomic, assign) CGPoint scaleBarOffset;
```

Comment:
I've been thinking about this further. `offset` in this context could be confusing, since the developer might assuming a positive value goes from left to right (like view coordinates) - it also depends on the order of the constraint attributes. 

These are really insets or margins.

With that in mind, perhaps we should consider these options:
- As a minimum, rename these properties to `scaleBarInsets` etc.
- Alternatively, how about a *single* `UIEdgeInsets` property that is used for *all* ornaments, `ornamentInsets`?
- iOS 9 introduced `UILayoutGuide` which could be used behind the scenes.

Reply:
As discussed, let's rename these to `scaleBarMargins` (and friends).

For history: we discussed having a single insets property, but it was felt this might be too restrictive.